.github/workflows/cut_release.yaml

```diff
@@ -12,17 +12,39 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - name: Get Tag
-        id: get_tag
+      - name: Get New Tag
```

Comment:
(nit) now that we can travel both: to the future and to the past, perhaps "desired tag" would better capture the intent